Src/FluentAssertions/Common/ExpressionExtensions.cs

```diff
@@ -46,7 +46,7 @@ internal static class ExpressionExtensions
                 throw new ArgumentNullException(nameof(expression), "Expected a property expression, but found <null>.");
             }
 
-            var memberInfo = AttemptToGetMemberInfoFromCastExpression(expression) ??
+            MemberInfo memberInfo = AttemptToGetMemberInfoFromCastExpression(expression) ??
```

Comment:
According AV1520 `RepositoryFactory.Get()` can use var. I think, here the type is clear, too.

Reply:
Debatable. I'd prefer an explicit type here.

Reply:
Can you give a reason? (I would like to understand. )
"MemberInfo" is part of the method name.
What is the difference to the repository sample?

Reply:
IMO, a variable is similar to a relationship/association in UML. It has a type and a role that type plays in the relationship. When there's no need to differentiate or when the type is obvious, `var` is fine. Otherwise, I prefer both.